doc/LIMITS.md

```diff
@@ -18,52 +18,51 @@ In order to prevent excessive resource consumption by a libp2p node it's importa
 
 ## Connection limits
 
-It's possible to limit the amount of incoming and outgoing connections a node is able to make.  When this limit is reached and an attempt to open a new connection is made, existing connections may be closed to make room for the new connection.
+It's possible to limit the amount of incoming and outgoing connections a node is able to make. When this limit is reached and an attempt to open a new connection is made, existing connections may be closed to make room for the new connection.
 
 We can also limit the number of connections in a "pending" state. These connections have been opened by a remote peer but peer IDs have yet to be exchanged and/or connection encryption and multiplexing negotiated. Once this limit is hit further connections will be closed unless the remote peer has an address in the [allow list](#allowdeny-lists).
 
 ```js
 const node = await createLibp2pNode({
   connectionManager: {
     /**
-     * The total number of connections allowed to be open at one time
+     * The total number of incoming connections allowed to be open at one time
      */
-    maxConnections: 200,
+    maxIncomingConnections: 200,
 
     /**
-     * If the number of open connections goes below this number, the node
+     * If the number of open outgoing connections goes below this number, the node
      * will try to connect to nearby peers from the peer store
      */
-    minConnections: 20,
+    minOutgoingConnections: 20,
 
     /**
      * How many connections can be open but not yet upgraded
      */
-    maxIncomingPendingConnections: 10
-  }
-})
+    maxIncomingPendingConnections: 10,
+  },
+});
```

Comment:
Do we want to lint the Typescript code examples in our docs? That could be a good feat to add to https://github.com/libp2p/js-libp2p/pull/1505